src/Controls/tests/UITests/Tests/Issues/_IssuesUITest.cs

```diff
@@ -9,15 +10,41 @@ public abstract class _IssuesUITest : UITestBase
 
 		protected override void FixtureSetup()
 		{
-			base.FixtureSetup();
-			NavigateToIssue(Issue);
+			int retries = 0;
+			while (true)
+			{
+				try
+				{
+					base.FixtureSetup();
+					NavigateToIssue(Issue);
+					break;
+				}
+				catch (Exception)
+				{
+					if (retries++ < 1)
+					{
+						TestContext.Error.WriteLine("There was a problem with the FixtureSetup retrying");
```

Comment:
Not an exact code suggestion, but could the error message be something like this? It will help devs understand why they're seeing a warning and what's happening next. Ultimately we'd presumably want to have zero of these, yet they are random, so very hard to detect and diagnose, so any extra info we can print out when it happens should be helpful.

```c#
						TestContext.Error.WriteLine($"The FixtureSetup threw an exception. Attempt {retries}/{MaxRetries}.\r\nException details: {e.ToString()}");
```

Reply:
Sure, thanks for the suggestion. I will change it up.